tutorials/introductory/lifecycle.py

```diff
@@ -17,15 +17,17 @@
     <https://pbpython.com/effective-matplotlib.html>`_
     by Chris Moffitt. It was transformed into this tutorial by Chris Holdgraf.
 
-A note on the Object-Oriented API vs. Pyplot
-============================================
+A note on the explicit vs. implicit interfaces
+==============================================
 
-Matplotlib has two interfaces. The first is an object-oriented (OO)
-interface. In this case, we utilize an instance of :class:`axes.Axes`
-in order to render visualizations on an instance of :class:`figure.Figure`.
+Matplotlib has two interfaces. For an explanation of the tradeoffs between the
+implicit and explicit interfaces see :ref:`api_interfaces`.
 
-The second is based on MATLAB and uses a state-based interface. This is
-encapsulated in the :mod:`.pyplot` module. See the :doc:`pyplot tutorials
+The first is an explicit object-oriented (OO) interface. In this case, we
+utilize an instance of :class:`axes.Axes` in order to render visualizations on
+an instance of :class:`figure.Figure`.  The second is based on MATLAB and uses
```

Comment:
This seems a bit jargony with the use of "render" here. My assumption for the audience is someone pretty new to MPL and interested in "plotting", "visualizing", and "showing" not necessarily rendering. So, perhaps one of those words instead here?